demo/full/styles/style.css

```diff
@@ -409,3 +573,107 @@ body {
 .time-indicator {
   height: 100%;
 }
+
+.nav-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 15px 0 15px;
+  background-color: #222424;
+  color: #ffffff;
+}
+
+.nav-header a {
+  color: #ffffff;
+}
+
+.header-links-buttons {
+  display: flex;
+  align-items: center;
+}
+
+.header-links-buttons > * {
+  margin: 3px;
+}
+
+.github-button {
+  margin: 5px;
+}
+
+.drm-settings {
+  background: #ffffff;
+  padding: 14px 14px 2px 14px;
+  margin: 12px 0 12px 0;
+  border: solid 1px #a9a9a9;
+}
+
+.drm-settings .input {
+  border: solid 1px #a9a9a9;
+}
+
+.white-select select {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Expected selector ".white-select select" to come before selector ".knob-value > select:focus" ([css:S4664](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4664))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXXLiJtashydt7kNvQB&open=AWXXLiJtashydt7kNvQB)

[ISSUE_KEY]: AWXXLiJtashydt7kNvQB